qa/tasks/mgr/dashboard/helper.py

```diff
@@ -226,23 +232,23 @@ def _request(cls, url, method, data=None, params=None):
                 # Output response for easier debugging.
                 log.error("Request response: %s", cls._resp.text)
             content_type = cls._resp.headers['content-type']
-            if content_type == 'application/json' and cls._resp.text and cls._resp.text != "":
+            if content_type.endswith('json') and cls._resp.text and cls._resp.text != "":
```

Comment:
Be careful here, [`Content-Type`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) could be:
```
Content-Type: text/html; charset=UTF-8
```
It's better IMHO to use regex here:
```py
re.match(r'^application/.*json', ...
```